plugins/ptp_operator/metrics/manager.go

```diff
@@ -164,7 +164,7 @@ func (p *PTPEventManager) GetPTPEventsData(state ptp.SyncState, ptpOffset int64,
 		return nil
 	}
 	// /cluster/xyz/ptp/CLOCK_REALTIME this is not address the event is published to
-	eventSource := fmt.Sprintf(p.resourcePrefix, p.nodeName, source)
+	eventSource := fmt.Sprintf(p.resourcePrefix, p.nodeName, fmt.Sprintf("/%s", source))
```

Comment:
would this work better?
```
eventSource := fmt.Sprintf(p.resourcePrefix, p.nodeName, "/", source)
```

Reply:
resourcePrefix         = "/cluster/node/%s%s" with two parameter  only
So it won't work